Source/cmake/FindGLibCompileResources.cmake

```
@@ -0,0 +1,84 @@
+pkg_get_variable(GLIB2_PREFIX glib-2.0 prefix)
+find_program(
+    GLIB_COMPILE_RESOURCES_EXE
+    NAMES glib-compile-resources
+    HINTS ${GLIB2_PREFIX}
+    REQUIRED
+)
+execute_process(
+    COMMAND ${GLIB_COMPILE_RESOURCES_EXE} --version
+    OUTPUT_VARIABLE glib_compile_resources_version
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+message(STATUS "Found glib-compile-resources (required): "
+    ${GLIB_COMPILE_RESOURCES_EXE}
+    " (${glib_compile_resources_version})")
+if (${glib_compile_resources_version} VERSION_LESS 2.77)
+    set(glib_compile_resources_has_depfile_bug TRUE)
+    message(STATUS "  Needs workaround for depfiles (will be used automatically)")
+else ()
+    set(glib_compile_resources_has_depfile_bug FALSE)
+    message(STATUS "  Doesn't need workarounds")
```

I think we do not need to inform about this, no need to make the find-module chatty.

```suggestion
else ()
    set(glib_compile_resources_has_depfile_bug FALSE)
```